controllers/C_Document.class.php

```diff
@@ -202,6 +202,16 @@ public function upload_action_process()
 
         if (is_numeric($_POST['category_id'])) {
             $category_id = $_POST['category_id'];
+        } else {
+            $category_id = 1;
+        }
+
+        // ensure user has access to the category that is being uploaded to
```

Comment:
likely need logic to skip this when services (ie. Direct) are uploading through this function